a/attrs/attrs_ubi_8.4.sh

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : gobuffalo/attrs
+# Version               : v0.0.0-20190224210810-a9411de4debd
+# Source repo           : https://github.com/gobuffalo/attrs.git
+# Tested on             : UBI 8.4
+# Language              : GO
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Vikas . <kumar.vikas@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=a9411de4debd
+else
+  export VERSION=$1
+fi
+if [ -d "attrs" ] ; then
+  rm -rf attrs
+fi
+
+# Dependency installation
+dnf install -y git gcc
```

Comment:
do we need gcc? in general it would be good to install only the minimal set of dependencies needed to build and test the package

Reply:
gcc is required to build the package:
```
cgo: exec gcc: exec: "gcc": executable file not found in $PATH
FAIL    github.com/gobuffalo/attrs [build failed]
```